CRP.Mvc/Views/Transaction/CheckoutNew.cshtml

```diff
@@ -150,44 +150,48 @@
     }
         @foreach (var qs in Model.Item.QuestionSets.Where(a => a.TransactionLevel).OrderBy(a => a.Order))
         {
-            <span id='@Html.Encode(qs.Id)'>
-                @{
-                    var questionSetName = qs.QuestionSet.Name;
-                    if (questionSetName == "Contact Information")
-                    {
-                        questionSetName = questionSetName + " / Billing Information";
-                        showDonationLink = true;
-                    }
-                    else
+            <div class="TransactionContainer">
+                <span id='@Html.Encode(qs.Id)'>
+                    @{
+                        var questionSetName = qs.QuestionSet.Name;
+                        if (questionSetName == "Contact Information")

```

Comment:
@jSylvestre if this works I'm fine with it but I do like the Ordinal string.Equals checks just to clarify what kind of string compare you are using.